cactusbot/services/beam/handler.py

```diff
@@ -35,10 +35,8 @@ def __init__(self, channel):
         }
 
         self.liveloading_events = {
-            "channel": {
```

Comment:
Want to keep `channel`... there are events with the same name for `user` and `channel`. (`channel:id:follow`/`user:id:follow`, for example.)